src/styles/_catalog.scss

```diff
@@ -1,10 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (min-width: 487px) {
+    grid-template-columns: 100%;
```

Comment:
create variable with width: 200px
and use grid-template-columns: repeat(2 || 3 || 4, $card-width);